src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd

```diff
@@ -0,0 +1,191 @@
+@echo off
+setlocal
+
+set Id=%1
+set IdMaster=Master
+set WcfTestDir=c:\WCFTest
+set RepoHome=c:\git
```

Comment:
It's latter. When you create the first repo on a machine, there is no repo that contains this script. You have to copy it from somewhere else. It's chicken and egg problem 😄.